velocity/tool/src/java/org/sakaiproject/cheftool/VelocityPortletPaneledAction.java

```diff
@@ -1178,6 +1178,19 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res) throws Ser
 		super.doGet(req, res);
 	}
 
+	// Set up RunData if it's not already set up    
+
+        protected void checkRunData(HttpServletRequest req)
```

Comment:
Chuck does it make sense that any request without velocity ATTR_PARAMS should be initialized like this without any check?